src/Microsoft.ML/CSharpApi.cs

```diff
@@ -106,18 +106,6 @@ public void Add(Microsoft.ML.Models.AnomalyDetectionEvaluator input, Microsoft.M
                 _jsonNodes.Add(Serialize("Models.AnomalyDetectionEvaluator", input, output));
             }
 
-            public Microsoft.ML.Models.AnomalyPipelineEnsemble.Output Add(Microsoft.ML.Models.AnomalyPipelineEnsemble input)
-            {
-                var output = new Microsoft.ML.Models.AnomalyPipelineEnsemble.Output();
```

Comment:
>feature [](start = 39, length = 7)

Is it possible to reduce the column count to 80 or at most 120 chars?  It's hard to read.